config/locales/flood_risk_engine/enrollments/steps/_confirmation.en.yml

```diff
@@ -0,0 +1,30 @@
+---
+en:
+  flood_risk_engine:
+    enrollments:
+      steps:
+        confirmation:
+          heading: Registration complete
+          heading_h2: What happens next
+          heading_h2_1: If your activity changes
+          para_text: 'Your reference is:'
+          email_sent: We've sent a confirmation email to %{email}
+          para_text2: You can begin work on the activity covered by your exemptions.
+          para_text3: Exemptions don’t expire, except for those for dredging which have time limits.
+                      If want to do the same activity again you must register a new exemption.
+          para_text4: We may check to see if your activity matches the conditions and
+            description of your exemption.
+          para_text5: You can check the conditions and description of your exemption
```

Comment:
I think:

para_text5: "You can check the conditions and description of your exemption in the %{link}."

is a better pattern